transformer_engine/common/common.h

```diff
@@ -50,6 +50,10 @@ inline bool is_delayed_tensor_scaling(const NVTEScalingMode &mode) {
 
 inline bool is_mxfp_scaling(const NVTEScalingMode &mode) { return mode == NVTE_MXFP8_1D_SCALING; }
 
+inline bool is_nvfp_scaling(const NVTEScalingMode &mode) {
+  return mode == NVTE_FWD_NVFP4_BWD_MXFP8_SCALING;
+}
```

Comment:
`NVTE_FWD_NVFP4_BWD_MXFP8_SCALING` is a very bad name: (see https://github.com/NVIDIA/TransformerEngine/pull/1779#discussion_r2220771240). I think we could make it clearer by blindly find-replacing with `NVTE_NVFP4_SCALING`.